eventuate-common-spring-jdbc/src/test/java/io/eventuate/common/spring/jdbc/SqlDialectIntegrationTest.java

```diff
@@ -113,6 +125,23 @@ public void testCurrentTime() throws InterruptedException {
     Assert.assertTrue(dbTime < javaTime2);
   }
 
+  @Test
+  public void testEventsPrimaryKeyColumn() throws SQLException {
+    testPrimaryKeyColumn("events", useDbId ? EventuateCommonJdbcOperations.EVENT_AUTO_GENERATED_ID_COLUMN : "event_id");
```

Comment:
Shouldn't column names like "id" and "event_id" be defined as constants like `EVENT_AUTO_GENERATED_ID_COLUMN`?

Reply:
ok